frontend/demo/component/button/button-focus.ts

```diff
@@ -0,0 +1,30 @@
+import 'Frontend/demo/init'; // hidden-full-source-line
+
+import { html, LitElement, customElement, css } from 'lit-element';
+import '@vaadin/vaadin-button/vaadin-button';
+import { applyTheme } from 'Frontend/generated/theme';
+
+@customElement('button-focus')
+export class Example extends LitElement {
+  constructor() {
+    super();
+    // Apply custom theme (only supported if your app uses one)
+    applyTheme(this.shadowRoot);
+  }
+
+  static get styles() {
+    return css`
+      vaadin-button {
+        outline: #8dbbf6 solid 2px;
+      }
+    `;
+  }
+
+  render() {
+    return html`
+      <!-- tag::snippet[] -->
+      <vaadin-button>Perform Action</vaadin-button>
```

Comment:
```suggestion
      <vaadin-button>Perform action</vaadin-button>
```

Use "Sentence case” for example UI texts.